### DISCUSSION TITLE
code untuk antrol bukan 200 bang, tapi 1

### DISCUSSION THREAD

**Discussion by MasruhChoeroni** [virusphp/service-bridging#7] [General]
code untuk antrol bukan 200 bang, tapi 1

**Comment by virusphp** [comment]
selamat malam terimakasih atas pemberitahuannya
untuk code antrol awalnya emang 200 mas untuk code 1 itu baru2 di release sama bpjs kemungkinan mreka salah nulis code atau documentasi karena awalnya code adalah 200 dan beberapa endpoint masih menggunakan code 200 seperti 

jadwal dokter
![image](https://user-images.githubusercontent.com/17744762/188686063-d9d41e05-c39a-4449-9d76-8fc9e75dc1be.png)

update jadwal dokter 
![image](https://user-images.githubusercontent.com/17744762/188686249-357fc5aa-9f53-473d-996f-8aafc5ac8da2.png)

tambah antrean 
![image](https://user-images.githubusercontent.com/17744762/188686333-6dee18f8-bdb6-4106-a059-5951993ade79.png)

batal antrean 
![image](https://user-images.githubusercontent.com/17744762/188686410-6ff905e7-d2d3-4e5a-add6-93ed9970df74.png)

dan beberapa lainnya masih menggunakan 200 mas
